esmvaltool/cmorizers/obs/cmor_config/GHCN_CAMS.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Please rename the file `GHCN-CAMS`.
Note this is not required for the .py file below.
(see other examples, like `ERA-Interim`).

Reply:
Done